packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -72,13 +90,29 @@ def __post_init__(self):
         # channels are not thread safe, what about python?
         self._channel_pool = aio_pika.pool.Pool(self._get_channel, max_size=10)
 
+    async def rpc_initialize(self) -> None:
+        # TODO: to SAN: not sure that we always want to setup RPC connection
+        self._rpc_connection = await aio_pika.connect_robust(
+            self.settings.dsn, client_properties={"connection_name": f"rpc.{uuid4()}"}
```

Comment:
why is the connection name different than the ones of the usual client? when looking into the rabbitmq monitoring tool, I'd be happy to know the connection of which service that is. and I guess you have one connection per client right? not sure the uuid4 is needed here.
```python
self._rpc_connection = await aio_pika.connect_robust(
            self.settings.dsn, client_properties={"connection_name": f"{self.client_name}.rpc.{uuid4()}"}
        )
```

Reply:
👍 will use your suggestion.

The UUID will be needed since it needs to distinguish between various instance of the client running on different hosts.

Reply:
what about using the hostname for this? it is more useful than a random UUID